playbooks/preflight-check.yml

```diff
@@ -84,8 +86,30 @@
       when:
         - fs_check.stdout_lines
       with_items: "{{ fs_check.stdout_lines }}"
-  tags:
-    - leap-fs-check
+      tags:
+        - leap-fs-check
+
+    - name: Gather lxc filesystem used percentages
+      shell: |
+        df -h {{ lxc_container_directory | default('/var/lib/lxc') }} | tail -n +2 | sed s/%//g | awk '{if($5 > 50) print "Alert: "$5 "% full - "$0}'
+      register: lxc_fs_check
+      when:
+        - inventory_host in groups["os-infra_hosts"]
+        - enable_lxc_fs_check | default(true) | bool
+
+    # RLM-1371 detect if enough space for rootfs backup
+    - name: Check if lxc filesystem is greater than 50% full
+      debug:
+        msg: >-
+          The LXC filesystem(s) have been found to be greater than 50% full.
+          Details: {{ item.split() | join(' ') }}
```

Comment:
Might need to wrap the brackets in quotes, the syntax checker in VIM seemed to not like this line.

Reply:
Cool, let me fix it really quick